src/descriptor/policy.rs

```diff
@@ -444,7 +452,7 @@ pub struct Policy {
     /// Type of this policy node
     #[serde(flatten)]
     pub item: SatisfiableItem,
-    /// How a much given PSBT already satisfies this polcy node **(currently unused)**
+    /// How a much a given PSBT already satisfies this policy node in terms of signatures
```

Comment:
small typo here.. should read "How much a given PSBT..."

Reply:
lol, sorry, my brain can process one " a " at a time